src/app/base/components/dropdown/dropdown.component.html

```diff
@@ -5,6 +5,7 @@
         <button #dropdownOpenButton type="button" *ngIf="dropdownOptions.buttonVersion == 'default'"
             class="inline-flex rounded-md border shadow-sm px-4 items-center text-xs font-semibold cursor-pointer focus:ring-offset-2 focus:ring-offset-gray-400"
             id="menu-button" aria-expanded="true" aria-haspopup="true" [disabled]="dropdownOptions.isDisabled"
+            [ngStyle]="dropdownOptions.maxWidth? { 'max-width': dropdownOptions.maxWidth } : {}"
```

Comment:
the width of the options should also be applied to the options (at least if the comment says so)